lib/openscap_parser/rule.rb

```diff
@@ -16,19 +16,19 @@ def severity
     end
 
     def title
-      @title ||= @rule_xml.at_css('title').children.first.text
+      @title ||= @rule_xml.at_css('title')&.text
     end
 
     def description
-      rule_node ||= @rule_xml.at_css('description')
-      @description ||= newline_to_whitespace(rule_node.text) if rule_node && rule_node.text
+      @description ||= newline_to_whitespace(
+        @rule_xml.at_css('description')&.text
+      )
     end
 
     def rationale
-      rationale_node ||= @rule_xml.at_css('rationale')
-      @rationale ||= newline_to_whitespace(rationale_node.children.text) if rationale_node &&
-        rationale_node.children &&
-        rationale_node.children.text
+      @rationale ||= newline_to_whitespace(
+        @rule_xml.at_css('rationale')&.text
+      )
```

Comment:
We should think about creating a helpers module for things like the `newline_to_whitespace` function, but I'm not including it in this PR.

Reply:
I agree, when we find out we need `newline_to_whitespace` in several places, we can extract it into a separate module.